picard/ui/metadatabox.py

```diff
@@ -501,8 +501,9 @@ def _update_tags(self):
                 tag_diff.objects += 1
 
         all_tags = set(list(orig_tags.keys()) + list(new_tags.keys()))
-        tag_names = COMMON_TAGS + \
-            sorted(all_tags.difference(COMMON_TAGS),
+        common_tags = [tag for tag in COMMON_TAGS if file.supports_tag(tag)]
+        tag_names = common_tags + \
```

Comment:
Use parentheses for line continuation?

Reply:
What about this?

```python
tag_names = common_tags + sorted(all_tags.difference(common_tags),
                                 key=lambda x: display_tag_name(x).lower())
```

That's probably even nicer.